pkg/apis/application/v1alpha1/types.go

```diff
@@ -1067,7 +1062,7 @@ func (repo *Repository) GetHelmCreds() helm.Creds {
 func getCAPath(repoURL string) string {
 	if git.IsHTTPSURL(repoURL) {
 		if parsedURL, err := url.Parse(repoURL); err == nil {
-			if caPath, err := cert.GetCertBundlePathForRepository(parsedURL.Host); err != nil {
+			if caPath, err := cert.GetCertBundlePathForRepository(parsedURL.Host); err == nil {
```

Comment:
@jannfis can you please confirm this is correct change?

Reply:
This should work when Helm expects a path to a CA file - if it expects the RAW certificate data, `GetCertificateForConnect()` would be the correct function.